File: PYTHON PROGRAMMING/Test8.py
# 8. SWAPPING OF TWO VARIABLES' VALUES WITHOUT USING A THIRD VARIABLE
input1 = int(input("Enter Input1: "))
input2 = int(input("Enter Input2: "))

print(f"Current Input1 Value: {input1}")
print(f"Current Input2 Value: {input2}")

input1 = input1 + input2
input2 = input1 - input2
input1 = input1 - input2

print(f"Updated Input1 Value: {input1}")
print(f"Updated Input2 Value: {input2}")

File: JAVASCRIPT PROGRAMMING/Test8.js
"use strict";
// 8. SWAPPING OF TWO VARIABLES' VALUES WITHOUT USING A THIRD VARIABLE
const swap = (ip1, ip2) => {
    console.log(`Current Num1 Value: ${ip1}`);
    console.log(`Current Num1 Value: ${ip2}`);
    ip2 = ip1 + ip2;
    ip1 = ip2 - ip1;
    ip2 = ip2 - ip1;
    console.log(`Updated Value Of Num1: ${ip1}`);
    console.log(`Updated Value Of Num2: ${ip2}`);
};
swap(20, 30);